curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-71-ordered-fractions.md

```diff
@@ -8,45 +8,88 @@ dashedName: problem-71-ordered-fractions
 
 # --description--
 
-Consider the fraction, `n`/`d`, where `n` and `d` are positive integers. If `n`&lt;`d` and HCF(`n`,`d`)=1, it is called a reduced proper fraction.
+Consider the fraction, $\frac{n}{d}$, where `n` and `d` are positive integers. If `n` &lt; `d` and highest common factor, ${{HCF}(n, d)} = 1$, it is called a reduced proper fraction.
 
 If we list the set of reduced proper fractions for `d` ≤ 8 in ascending order of size, we get:
 
-<div style='text-align: center;'>1/8, 1/7, 1/6, 1/5, 1/4, 2/7, 1/3, 3/8, <strong>2/5</strong>, 3/7, 1/2, 4/7, 3/5, 5/8, 2/3, 5/7, 3/4, 4/5, 5/6, 6/7, 7/8</div>
+<div style='text-align: center;'>$\frac{1}{8}$, $\frac{1}{7}$, $\frac{1}{6}$, $\frac{1}{5}$, $\frac{1}{4}$, $\frac{2}{7}$, $\frac{1}{3}$, $\frac{3}{8}$, $\frac{\textbf2}{\textbf5}$, $\frac{3}{7}$, $\frac{1}{2}$, $\frac{4}{7}$, $\frac{3}{5}$, $\frac{5}{8}$, $\frac{2}{3}$, $\frac{5}{7}$, $\frac{3}{4}$, $\frac{4}{5}$, $\frac{5}{6}$, $\frac{6}{7}$, $\frac{7}{8}$</div>
```

Comment:
```suggestion
$\frac{1}{8}$, $\frac{1}{7}$, $\frac{1}{6}$, $\frac{1}{5}$, $\frac{1}{4}$, $\frac{2}{7}$, $\frac{1}{3}$, $\frac{3}{8}$, $\frac{\textbf2}{\textbf5}$, $\frac{3}{7}$, $\frac{1}{2}$, $\frac{4}{7}$, $\frac{3}{5}$, $\frac{5}{8}$, $\frac{2}{3}$, $\frac{5}{7}$, $\frac{3}{4}$, $\frac{4}{5}$, $\frac{5}{6}$, $\frac{6}{7}$, $\frac{7}{8}$
```
I think it looks a little nicer without the div, since it gets quite close to the text below:
![image](https://user-images.githubusercontent.com/15801806/117258380-33d14680-ae4d-11eb-88a7-a8a5f9eb9d4f.png)

and without the div:
![image](https://user-images.githubusercontent.com/15801806/117258306-1b612c00-ae4d-11eb-8cd0-0586b53452a7.png)

It's no longer centered, mind.  What do you reckon?

Reply:
I've played a bit with it, and there's way to keep it centered and with even spacing using MathJax. `$$` delimiter will make it styled as displayed equation, instead of in-line. What do you think?
```suggestion
$${\frac{1}{8}}, {\frac{1}{7}}, {\frac{1}{6}}, {\frac{1}{5}}, {\frac{1}{4}}, {\frac{2}{7}}, {\frac{1}{3}}, {\frac{3}{8}}, {\frac{\textbf2}{\textbf5}}, {\frac{3}{7}}, {\frac{1}{2}}, {\frac{4}{7}}, {\frac{3}{5}}, {\frac{5}{8}}, {\frac{2}{3}}, {\frac{5}{7}}, {\frac{3}{4}}, {\frac{4}{5}}, {\frac{5}{6}}, {\frac{6}{7}}, {\frac{7}{8}}$$
```
![fractions](https://user-images.githubusercontent.com/60067306/117289228-a4d42680-ae6c-11eb-8813-0dec45d00a5d.png)